src/libraries/System.Linq/src/System/Linq/Except.cs

```diff
@@ -37,8 +37,27 @@ public static IEnumerable<TSource> Except<TSource>(this IEnumerable<TSource> fir
             return ExceptIterator(first, second, comparer);
         }
 
+        /// <summary>
+        /// Produces the set difference of two sequences according to a specified key selector function.
+        /// </summary>
+        /// <typeparam name="TSource">The type of the elements of the input sequence.</typeparam>
+        /// <typeparam name="TKey">The type of key to identify elements by.</typeparam>
+        /// <param name="first">An <see cref="IEnumerable{TSource}" /> whose keys that are not also in <paramref name="second"/> will be returned.</param>
+        /// <param name="second">An <see cref="IEnumerable{TKey}" /> whose keys that also occur in the first sequence will cause those elements to be removed from the returned sequence.</param>
+        /// <param name="keySelector">A function to extract the key for each element.</param>
+        /// <returns>A sequence that contains the set difference of the elements of two sequences.</returns>
         public static IEnumerable<TSource> ExceptBy<TSource, TKey>(this IEnumerable<TSource> first, IEnumerable<TKey> second, Func<TSource, TKey> keySelector) => ExceptBy(first, second, keySelector, null);
 
+        /// <summary>
+        /// Produces the set difference of two sequences according to a specified key selector function.
+        /// </summary>
+        /// <typeparam name="TSource">The type of the elements of the input sequence.</typeparam>
+        /// <typeparam name="TKey">The type of key to identify elements by.</typeparam>
+        /// <param name="first">An <see cref="IEnumerable{TSource}" /> whose keys that are not also in <paramref name="second"/> will be returned.</param>
+        /// <param name="second">An <see cref="IEnumerable{TKey}" /> whose keys that also occur in the first sequence will cause those elements to be removed from the returned sequence.</param>
```

Comment:
This isn't grammatical I thnk? How about this or similar?
```suggestion
        /// <param name="first">An <see cref="IEnumerable{TSource}" /> that will form the return sequence after removing any elements that share keys with elements in the<paramref name="second"/> sequence.</param>
        /// <param name="second">An <see cref="IEnumerable{TKey}" /> that is used to identify elements in the <paramref name="first"/> sequence that should not be present in the returned sequence.</param>
```
and the others.

Reply:
My thoughts exactly, however this is merely copying the grammar from the existing [`Except` method documentation](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Linq/Enumerable.xml):
```xml
        <param name="first">An <see cref="T:System.Collections.Generic.IEnumerable`1" /> whose elements that are not also in <paramref name="second" /> will be returned.</param>
        <param name="second">An <see cref="T:System.Collections.Generic.IEnumerable`1" /> whose elements that also occur in the first sequence will cause those elements to be removed from the returned sequence.</param>
```
My aim was to go for uniformity and then attempt to fix grammar issues for everything (preferably once we have established a single source of truth for docs).

Reply:
Fine by me